package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@protobufjs/utf8": "^1.1.0",
-    "binaryen": "^55.0.0-nightly.20181129",
+    "binaryen": "55.0.0-nightly.20181129",
```

Comment:
I ran into an issue once where just editing this line didn't work as expected, while `npm install --save-exact binaryen@55.0.0-nightly.20181129" did. Not sure if that was a random error, but it might be that this also requires an update in package-lock or something that only `--save-exact` applies to actually pin it.

Reply:
got it!

Reply:
That's behaviour of npm (I mean verbose cli) is really contrast with `yarn` which `yarn add <package>` install and save to package.json in the same time exactly as you expected